app/services/licences/fetch-licence.service.js

```diff
@@ -0,0 +1,51 @@
+'use strict'
+
+/**
+ * Fetches data needed for the bill run page which includes a summary for each bill linked to the bill run
+ * @module FetchLicenceService
+ */
+
+const LicenceModel = require('../../models/licence.model.js')
+
+/**
+ * Fetch the licence data from the water schema
+ *
+ * Was built to provide the data needed for the '/licences/{id}' page
+ *
+ * @param {string} id The UUID for the licence to fetch
+ *
+ * @returns {Promise<Object>} the data needed to populate the summary page
+ */
```

Comment:
Good or bad (we're still deciding!) but the intent of adding views that we then base our models on was to hide the confusion that all the schemas add and make it seem like we're dealing with just one DB/schema.

So, I'd suggest not worrying about referencing that here.

```suggestion
/**
 * Fetch the matching licence and return data needed for the view licence page
 *
 * @param {string} id The UUID for the licence to fetch
 *
 * @returns {Promise<Object>} the data needed to populate the view licence page and some elements of the summary tab
 */
```